run.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+echo "Migrating..."
+./logflare eval Logflare.Release.migrate
+echo "Migration done; Starting app"
+./logflare start --sname logflare
```

Comment:
would prefer to have not have a separate run script, seems unnecessary when we can put the migration and start command inside of the dockerfile image RUN command and chain them with `&&`.

Reply:
Also, more files = more maintenance

Reply:
I would rather keep it like this for now. 

It's one more small file that enable us to add more tasks if need be (potentially seeding?). 

Plus the RUN command would be run during the build process of docker so we would be running migrations during the building process. We could have a CMD that does it but not a big win in terms of readability.

Reply:
ah yes i was referring to the `CMD` directive, sorry for confusion.

Sure, no issue then. Seeding would be done as an elixir module, not through shell scripting.